darts/models/torch_forecasting_model.py

```diff
@@ -261,46 +265,63 @@ def _create_from_cls_and_kwargs(cls, kws):
 
     @random_method
     def fit(self,
-            series: TimeSeries,
-            val_series: Optional[TimeSeries] = None,
-            verbose: bool = False,
-            target_indices: Optional[List[int]] = None) -> None:
+            covariate_series: TimeSeries,
+            target_series: Optional[TimeSeries] = None,
+            val_series: Optional[Tuple[TimeSeries]] = None,
```

Comment:
How about splitting in two:
```
val_covariate_series: Optional[TimeSeries]
val_target_series: Optional[TimeSeries]
```
That would be more consistent with the first two arguments.

Reply:
I did it like this first but having a Tuple makes it more easy to check that the length provided is either None or 2 as opposed to having two separate args

Reply:
I would say that it's more important to have a good signature here, even if the check is slightly more complex behind the scenes.